CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ endif()
 
 project(migraphx)
 find_package(ROCM REQUIRED)
+find_package(nlohmann_json 3.8.0 REQUIRED)
```

Comment:
I am not sure if its best to put the version here, I believe there is some backwards compatibility, but I am not sure.